tests/test_cli.py

```diff
@@ -694,3 +694,37 @@ def query_server_side_effect(*args, **kwargs):
     ] == snapshot
 
     assert len(bat_calls) == 2
+
+
+def test_server_does_not_exist_error(runner_with_error, mocker):
+    mocker.patch(
+        "seagoat.cli.requests.get", side_effect=requests.exceptions.ConnectionError
+    )
+    query = "JavaScript"
+    result = runner_with_error.invoke(seagoat, [query])
```

Comment:
probably you need to make the same change here

Reply:
Yes, even if now the result is the same, the running server could fail for another reason